system-x/services/filesystem/src/main/java/software/tnb/filesystem/service/FileSystem.java

```diff
@@ -0,0 +1,11 @@
+package software.tnb.filesystem.service;
+
+import software.tnb.common.service.Service;
+
+import java.nio.file.Path;
+
+public abstract class FileSystem implements Service {
```

Comment:
Do you think `implements Service` is mandatory? what happend if you remove it? since we are not using beforeAll/afterAll

Reply:
I tried to remove it and looks it's mandatory